diff --git a/node_modules/electron-log/src/main/ElectronExternalApi.js b/node_modules/electron-log/src/main/ElectronExternalApi.js
index 45d8840..bf81982 100644
--- a/node_modules/electron-log/src/main/ElectronExternalApi.js
+++ b/node_modules/electron-log/src/main/ElectronExternalApi.js
@@ -6,11 +6,13 @@ const NodeExternalApi = require('../node/NodeExternalApi');
 
 class ElectronExternalApi extends NodeExternalApi {
   getAppName() {
+    let appName;
     try {
-      return electron.app?.name || electron.app?.getName();
-    } catch {
-      return super.getAppName();
+      appName = electron.app?.name || electron.app?.getName();
+    } catch (err) {
+      // fallback to default value below
     }
+    return appName || super.getAppName();
   }
 
   getAppUserDataPath(appName) {
@@ -18,11 +20,13 @@ class ElectronExternalApi extends NodeExternalApi {
   }
 
   getAppVersion() {
+    let appVersion;
     try {
-      return electron.app?.getVersion();
-    } catch {
-      return super.getAppVersion();
+      appVersion = electron.app?.getVersion();
+    } catch (err) {
+      // fallback to default value below
     }
+    return appVersion || super.getAppVersion();
   }
 
   getElectronLogPath() {
